src/app/learn/reference/configuration/index.html

```diff
@@ -37,7 +37,7 @@
         </li>
         <li>
           <p>
-            <stache-code>externals</stache-code> — Dynamically injects CSS and JavaScript files into SKY UX Host. You should have a specific use-case for an external. For example, Office Addins require their library to load through CDN in the head. The <stache-code>before</stache-code> and <stache-code>after</stache-code> sections of an external indicate whether to include the external resource before or after the default SKY UX Builder resources. The <stache-code>head</stache-code> property, which only applies to JS resources, indicates whether to inject elements within the HTML <stache-code>head</stache-code> element or just before the closing <stache-code>body</stache-code> tag.
+            <stache-code>externals</stache-code> — Dynamically injects CSS and JavaScript files into SKY UX Host for the <stache-code>skyux serve</stache-code> for the <stache-code>skyux build</stache-code> commands. You should have a specific use-case for an external. For example, Office Addins require their library to load through CDN in the head. The <stache-code>before</stache-code> and <stache-code>after</stache-code> sections of an external indicate whether to include the external resource before or after the default SKY UX Builder resources. The <stache-code>head</stache-code> property, which only applies to JS resources, indicates whether to inject elements within the HTML <stache-code>head</stache-code> element or just before the closing <stache-code>body</stache-code> tag.
```

Comment:
The "for" between the two commands should be an "or."

Reply:
Copy and paste error. Sorry. Fixing now...